x/slashing/errors.go

```diff
@@ -10,7 +10,7 @@ type CodeType = sdk.CodeType
 
 const (
 	// Default slashing codespace
-	DefaultCodespace sdk.CodespaceType = 10
+	DefaultCodespace sdk.CodespaceType = MsgRoute
```

Comment:
Hmm, although sharing the variable is better reading through this line doesn't make a lot of sense (what does the codespace have to do with the msg route) - maybe we should declare `const ModuleName = "slashing"` and assign both `MsgRoute` and `DefaultCodespace` to `ModuleName`, what do you think?

Reply:
++

Reply:
Just setting new strings that are capitalized instead, so as to differentiate.